deploy/complete/helm-chart/setup/values.yaml

```diff
@@ -120,7 +121,6 @@ grafana:
     - grafana-worldmap-panel
     - grafana-piechart-panel
     - btplc-status-dot-panel
-    - grafana-github-datasource
```

Comment:
Removal necessary as this plugin does not support arm64 and is not used by developers use case with MuShop

Reply:
👍 . I’ve never seen anyone request this either

Reply:
I was using to get stats on clones/forks. There's a new, but is out of context for MuShop deployment.